### DISCUSSION TITLE
How can I use a item from one collection in another? (Many to Many)

### DISCUSSION THREAD

**Discussion by matifanger** [directus/directus#16387] [Archive]
So I have a collection called "Gallery" and other called "Actors".

When I'm creating an item in "Gallery", let's say:
```js
{
    title: 'my title',
    slug: 'my-title',
    persons: How I can use any item previously created on "Actors" collection? (persons is an array of "Actors" as many to many)
}
```

I'm posting directly to the API rest.

**Comment by azrikahar** [comment]
You can use either the basic syntax (an array) or the detailed syntax (an object with create/update/delete props) as described here: https://docs.directus.io/reference/introduction.html#one-to-many-many-to-many

**Reply by matifanger** [reply]
The array syntax is not working for me, 
Im trying
```js
{
    title: 'my title',
    slug: 'my-title',
    persons: [1], // I mean it's the id, right?
}
```

**Reply by azrikahar** [reply]
Ah in this case the `1` here is the junction table id, not the actor id, so it would instead need to be nested for actor id:

```js
{
    title: 'my title',
    slug: 'my-title',
    persons: [
        {
            actors_id: 1 // "actors_id" is assumed as the junction table column for actors, so you should double check this in the junction table which should be `gallery_actors` if you didn't rename it.
        }
    ]
}
```

**Reply by matifanger** [reply]
Perfect! That is what I was looking for, thank you.
